scripts/generateThorLongtailTokens/generateThorLongtailTokens.ts

```diff
@@ -0,0 +1,82 @@
+import type { ChainReference } from '@shapeshiftoss/caip'
+import { ASSET_NAMESPACE, CHAIN_NAMESPACE, toAssetId } from '@shapeshiftoss/caip'
+import axios from 'axios'
+import fs from 'fs'
+import path from 'path'
+
+type Token = {
+  chainId: number
+  address: string
+  decimals: number
+  name: string
+  symbol: string
+  logoURI: string
+}
+
+type TokenList = {
+  name: string
+  logoURI: string
+  keywords: string[]
+  version: {
+    major: number
+    minor: number
+    patch: number
+  }
+  tokens: Token[]
+}
+
+const ethUrl =
+  'https://gitlab.com/thorchain/thornode/-/raw/4601fe74d5f990052e11d29b036dce5b727da990/common/tokenlist/ethtokens/eth_mainnet_latest.json'
+const avaxUrl =
+  'https://gitlab.com/thorchain/thornode/-/raw/4601fe74d5f990052e11d29b036dce5b727da990/common/tokenlist/avaxtokens/avax_mainnet_latest.json'
+const bscUrl =
+  'https://gitlab.com/thorchain/thornode/-/raw/4601fe74d5f990052e11d29b036dce5b727da990/common/tokenlist/bsctokens/bsc_mainnet_latest.json'
+
+const outputPath =
+  '../../src/lib/swapper/swappers/ThorchainSwapper/generated/generatedThorLongtailTokens.json'
+
+const axiosConfig = {
+  timeout: 10000,
+  headers: {
+    Accept: 'application/json',
+    'Content-Type': 'application/json',
+  },
+}
+
+export const generateThorLongtailTokens = async () => {
+  const thorService = axios.create(axiosConfig)
+  const ethResponse = await thorService.get<TokenList>(ethUrl)
+  const avaxResponse = await thorService.get<TokenList>(avaxUrl)
+  const bscResponse = await thorService.get<TokenList>(bscUrl)
```

Comment:
promise.all could be handy here

Reply:
oh i see this is not performance critical, disregard

Reply:
Did it anyway, might as well!

https://github.com/shapeshift/web/pull/5638/commits/f934129d2ff4c299ba286c9549ba9e24a96fdb5c